hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java

```diff
@@ -272,4 +272,12 @@ public static synchronized ServerOptions instance() {
                     disallowEmpty(),
                     "disable"
             );
+
+    public static final ConfigOption<Long> SLOW_QUERY_LOG_TIME_THRESHOLD =
+            new ConfigOption<>(
+                    "log.slow_query_threshold",
+                    "The threshold time(ms) of logging slow query.",
```

Comment:
append "0 means logging slow query is disabled."

Reply:
**Update:** I have fix the conflicts, could just pull it